src/main/kotlin/com/openlattice/search/PersistentSearchService.kt

```diff
@@ -100,9 +99,11 @@ class PersistentSearchService(private val hds: HikariDataSource, private val spm
                 EMAILS.name
         )
 
-        return "INSERT INTO ${PERSISTENT_SEARCHES.name} (${columns.joinToString(
-                ","
-        )}) VALUES ('${search.id}'::uuid, '${getUserAclKeyArray(connection)}', '${search.lastRead}', '${search.expiration}', " +
+        return "INSERT INTO ${PERSISTENT_SEARCHES.name} (${
+            columns.joinToString(
```

Comment:
minor, this should probably be a constant to avoid `joinToString` being called just to create a constant for every insert